src/matlab2tikz.m

```diff
@@ -3120,26 +3129,35 @@ function warnFacetedInterp(m2t)
         ['A 3D plot with "shader = faceted interp" is being produced.\n', ...
         'This may produce big and sluggish PDF files.\n', ...
         'See %s and Section 4.6.6 of the pgfplots manual for workarounds.'], ...
-        issueUrl(693, true));
+        issueUrl(m2t, 693, true));
 end
 % ==============================================================================
-function url = issueUrl(number, forOutput)
+function url = issueUrl(m2t, number, forOutput)
```

Comment:
I would pass `m2t.about.issues`, instead of the whole structure `m2t` (like it is done for the calls `clickableUrl` above).